### PR TITLE
mungegithub: Export metrics through Prometheus

### PR DESCRIPTION
**Pull request by apelisse**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/contrib/2070)

### CODE REVIEW

File: mungegithub/mungers/submit-queue.go [hunk 1]
```diff
@@ -150,6 +152,39 @@ type submitQueueMetadata struct {
...
+			Help: "Number of open pull-requests",
+		}),
+		QueuedPRs: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "submitqueue_queued_pullrequests_total",
+			Help: "Number of pull-requests queued",
+		}),
+		MergeCount: prometheus.NewCounter(prometheus.CounterOpts{
+			Name: "submitqueue_merge_total",
+			Help: "Number of merges done",
+		}),
```
Comment (apelisse) [inline mungegithub/mungers/submit-queue.go]: Can't we just do a "rate" for that?

**Comment by rmmh** [comment]
I think you forgot to check in the vendored code, here.

**Comment by apelisse** [comment]
Thanks!

**Pull request merged**